crates/typst-library/src/text/raw.rs

```diff
@@ -92,6 +93,11 @@ pub struct RawElem {
     /// )
     ///
     /// With `rg`, you can search through your files quickly.
+    /// This example searches the current directory recursively
+    /// for the text ``` Hello World```:
+    /// // (Note the leading space in the triple-backtick
```

Comment:
I think it's better to keep longer explanations like this out of the example and move it into the actual documentation prose.

Reply:
From what I have seen in the documentation, the example should also be the last part of the documentation, right? In that case, I lean towards adding this to the overall syntax section (i.e. [here](https://typst.app/docs/reference/text/raw/#syntax)), does that sound good?

Reply:
Yes, that would be great.